WooCommerce/Classes/POS/Presentation/Reusable Views/POSSendReceiptView.swift

```diff
@@ -28,13 +28,16 @@ struct POSSendReceiptView: View {
             }))
 
             VStack(alignment: .center, spacing: conditionalPadding(8)) {
-                TextField(Localization.textfieldPlaceholder, text: $textFieldInput)
+                TextField("",
+                          text: $textFieldInput,
+                          prompt: Text(Localization.textfieldPlaceholder).foregroundColor(.posOnDisabledContainer))
```

Comment:
TIL about this initializer, so the placeholder's style is accessible/formateable 👍